app/components/deadline_banner_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<% if CycleTimetable.show_apply_1_deadline_banner? %>
+  <%= govuk_notification_banner(title: "Important") do |notification_banner| %>
+    <%= notification_banner.slot(:heading, text: "Apply now to get on a course starting in the #{CycleTimetable.current_year} to #{CycleTimetable.next_year} academic year") %>
+    <p class="govuk-body">Courses can fill up at any time, so you should apply as soon as you can.</p>
+    <p class="govuk-body">If you’re applying for the first time since applications opened in <%= CycleTimetable.find_opens.strftime('%B %Y') %>, apply no later than 6pm on <%= apply_1_deadline %>.</p>
+    <p class="govuk-body">If your application did not lead to a place and you’re applying again, apply no later than 6pm on <%= apply_2_deadline %>.</p>
+  <% end %>
+<% elsif CycleTimetable.show_apply_2_deadline_banner? %>
+  <%= govuk_notification_banner(title: "Important") do |notification_banner| %>
+    <%= notification_banner.slot(:heading, text: "If you’re applying for the first time since applications opened in #{CycleTimetable.find_opens.strftime('%B %Y')}") %>
+    <p class="govuk-body">It’s no longer possible to apply for teacher training starting in the <%= "#{CycleTimetable.current_year} to #{CycleTimetable.next_year}" %> academic year.</p>
+    <p class="govuk-body">Come back from 9am on <%= find_reopens %> to find courses starting in the <%= "#{CycleTimetable.next_year} to #{CycleTimetable.next_year + 1}" %> academic year.</p>
```

Comment:
Shall we put these into the calendar as times?

Reply:
Yea i'm going to add a proper dates helper like we have on Apply but in a separate PR as i'm aware this needs to go out today